markdown/metadata.go

```diff
@@ -8,8 +8,15 @@ import (
 
 // Metadata is document metadata in the "front matter" of a Markdown document.
 type Metadata struct {
-	Title                       string `yaml:"title"`
-	IgnoreDisconnectedPageCheck bool   `yaml:"ignoreDisconnectedPageCheck"`
+	IgnoreDisconnectedPageCheck bool `yaml:"ignoreDisconnectedPageCheck"`
+
+	Title         string   `yaml:"title"`
+	Description   string   `yaml:"description"`
+	Category      string   `yaml:"category"`
+	CanonicalPath string   `yaml:"canonicalPath"`
+	ImagePath     string   `yaml:"imagePath"`
```

Comment:
Why imagePath instead of just image?

Reply:
@bobheadxi here's my reasoning: without `path` you cannot know if that's a path or an URL that you need to put in there.

Reply:
I think this can be either right? you can have a relative path from root (though this also depends on how one sets up the template), or a full URL (as recommended in e.g. the handbook so as to avoid bloating git history)

Given this ambiguity I think we should just match the standard metadata exactly

Reply:
Ack, moving to a full URL then, thanks!